web/modules/sheaf/src/core.ts

```diff
@@ -72,7 +72,8 @@ export class SheafCore {
   }
 
   private update(update: ViewUpdate) {
-    if (!update.docChanged && !update.selectionSet) return
+    // if (!update.docChanged && !update.selectionSet) return
```

Comment:
Why is this code commented out?

Reply:
Ah, oops. That's temporary. With the original check, moving your cursor at all would cause the FTML to get rerendered. For now, the editor state will only update if the document changes.

Later, that will need to be properly fixed, because if we add sourcemapping there will need to be a bunch of editor state that updates with scrolling, active lines, etc.